### ISSUE TITLE
Known issues, typescript?

### ISSUE THREAD

**Issue by shawnmclean** [electblake/rushstack-sst-playground#1]
> only yarn worked - I'm not sure why now, but pnpm did not work and rush only supports an old version of npm and I'd rather use node12 or 14

Was this referring to a typescript error when running `start`?

I'm trying to use pnpm and getting: `/common/temp/node_modules/.pnpm/typescript@4.4.2/node_modules/.bin/tsc: No such file or directory`

**Comment by kierans** [comment]
This is because of https://github.com/serverless-stack/serverless-stack/issues/854

**Comment by electblake** [comment]
yep - sst couldn't resolve using the pnpm cache module format (hoisted). Similarily, yarn2 only worked with using `nodeLinker: node-modules` which mirrors the classic `node_modules` structure vs. their pnp stetup.
